py-wacz/wacz/waczindexer.py

```diff
@@ -344,6 +342,8 @@ def generate_metadata(self, res, wacz):
             metadata["mainPageTS"] = res.date
 
         package_dict["metadata"] = metadata
-        package_dict["wacz_version"] = WACZ_VERSION
+        package_dict["software"] = {}
```

Comment:
Perhaps `wacz_version` should stay in the root, but `software` should be the version of the tool (py-wacz in this case) so it would be:
```
wacz_version: 1.0.0
software: py-wacz 0.3.0
```